pom.xml

```diff
@@ -53,6 +53,12 @@
             <version>1.6.4</version>
         </dependency>
 
+        <dependency>
+            <groupId>com.microsoft.azure</groupId>
+            <artifactId>azure</artifactId>
+            <version>1.23.0</version>
+        </dependency>
+
```

Comment:
Please ensure all tests pass with this verison

Reply:
fixed, all tests are passing.